doc-content/enterprise-only/openshift/template-deploy-start-proc.adoc

```diff
@@ -64,6 +56,21 @@ endif::DM[]
 :template_add_params: -p KIE_SERVER_HTTPS_SECRET=kieserver-app-secret
 endif::[]
 
+ifeval::["{context}"=="server-immutable-s2i"]
+:single_template_file!:
+:template_add_params: -p KIE_SERVER_HTTPS_SECRET=kieserver-app-secret
+ifdef::PAM[]
+:modifylink: environment-immutable-modify-proc
+:modifytype: fixed
+endif::PAM[]
+:template_name: <template-file-name>.yaml
+To deploy {what_deploying}, use one of the following template files:
+
+* If you want to configure the immutable {KIE_SERVER} to receive and respond to API messages using JMS, communicating with an AMQ server, use the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-kieserver-amq.yaml` template file.
```

Comment:
don't use "to receive and respond to API messages using JMS" it is confusing. Also, re-phrase the "communicating with AMQ server" sentence.